docs/operator-manual/applicationset/GoTemplate.md

```diff
@@ -74,6 +74,13 @@ All your templates must replace parameters with GoTemplate Syntax:
 
 Example: `{{ some.value }}` becomes `{{ .some.value }}`
 
+### Cluster Generators
+
+By activating Go Templating, `{{ .metadata }}` becomes an object.
+
+- `{{ metadata.labels.my-label }}` becomes `{{ index .metadata.labels "my-label" }}`
+- `{{ metadata.annotations.my/annotation }}` becomes `{{ index .metadata.annotations "my/annotation" }}`
+
```

Comment:
@crenshaw-dev , was wondering if we need to provide a new example for this section. 
Or we could put this section below git generator and relate to the example present in the git generators

Reply:
I think in this case I'd rather keep it as a simple "old format" vs. "new format" section. I'm not sure that there's anything to "demonstrate" with an example. The new syntax may be used anywhere the old syntax was used.